### DISCUSSION TITLE
Find some issues that are related to the programming languages that you are good at

### DISCUSSION THREAD

**Discussion by ZuxingGit** [ZuxingGit/pygments#7] [Ideas]
Find some issues that are realated to the programming languages that you are good at, so that we don't need to guess the syntax of sample code is right or wrong.

**Comment by Arseneme** [comment]
You're absolutely right

**Comment by trungtruong1** [comment]
So we are looking for how many new issues?

**Reply by ZuxingGit** [reply]
I will try to find a new one for myself.

**Reply by trungtruong1** [reply]
I agree, I'll find one for myself as well.
